lbz/configuration.py

```diff
@@ -1,13 +1,30 @@
+import json
 from abc import ABCMeta, abstractmethod
 from os import getenv
-from typing import Any, Callable, Generic, Optional, TypeVar
+from typing import Any, Callable, Generic, List, Optional, TypeVar, cast
 
 from lbz.aws_ssm import SSM
 from lbz.exceptions import ConfigValueParsingFailed, MissingConfigValue
 
 T = TypeVar("T")
 
 
+class ConfigParser:
+    @staticmethod
+    def split_by_comma(cfg: str) -> List[str]:
+        return cfg.split(",")
+
+    @staticmethod
+    def env_bool(cfg: str) -> bool:
+        if cfg.lower() in ("true", "1"):
+            return True
+        return False
+
+    @staticmethod
+    def load_jwt_keys(cfg: str) -> List[dict]:
```

Comment:
Also, it would be good to write simple tests for these parsers - it will simplify checking what is expected as an input/output 😉